src/jit/importer.cpp

```diff
@@ -7220,11 +7216,6 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 
                 GenTreePtr fptr = impImportLdvirtftn(thisPtr, pResolvedToken, callInfo);
 
-                if (compDonotInline())
-                {
-                    return callRetTyp;
-                }
-
```

Comment:
Looks like `impImportLdvirtftn` can fail, so don't we still need this check?

Reply:
The same as the first, we always about inlining before this check.

Reply:
Add an assert here too.

Reply:
Fixed